### PR TITLE
Deprecate the reference to a parent commit

### PR DESCRIPTION
**Pull request by lubosmj**
[noissue]

### CODE REVIEW

File: pulpcore/cli/ostree/repository.py [hunk 1]
```diff
@@ -193,19 +201,23 @@ def import_commits(
...
+    }
+
+    if pulp_ctx.has_plugin(PluginRequirement("ostree", max="2.0.0a3.dev")):
+        if not all((ref, parent_commit)) and any((ref, parent_commit)):
+            raise click.ClickException(
+                "Please specify both the ref and parent_commit if you want to add new child "
+                "commits to the existing repository"
+            )
+
+        kwargs.update({"parent_commit": parent_commit})
```
Comment (mdellweg) [inline pulpcore/cli/ostree/repository.py:221]:
```suggestion
        kwargs["parent_commit"] = parent_commit
```

File: pulpcore/cli/ostree/repository.py [hunk 2]
```diff
@@ -21,6 +22,7 @@
     label_select_option,
     list_command,
     name_option,
+    pulp_option,
```
Comment (mdellweg) [inline pulpcore/cli/ostree/repository.py:25]: Can you make sure, that the required pulp-cli version (0.12) contains `pulp_option` with the `needs_plugin`?

**Review by mdellweg — APPROVED** [review]
Code looks good to me.
Please check the dependencies before merging.

File: pulpcore/cli/ostree/repository.py (hunk 2 above)
Comment (lubosmj) [reply pulpcore/cli/ostree/repository.py:25]: The argument was added in version 0.13. I am bumping the requirement.

**Pull request merged**